### ISSUE TITLE
Not able to change default workspace directory

### ISSUE THREAD

**Issue by Marlene163** [exercism/exercism#6174]
I changed Operating system from Linux to Windows (Linux is still on this computer) and tried to download exercism again (for Windows). Everything works, but I can't find the workspace directory.

<img width="580" alt="image" src="https://user-images.githubusercontent.com/65309789/147574657-40adc6c3-8403-4c90-9dc1-6f10d2aedf99.png">

I think that exercism puts the workspace Exercism-Folder in my users directory in Linux but not Windows.
I really don't know how to fix this. Help is very welcome.

**Comment by tejasbubane** [comment]
@Marlene163 `exercism workspace` command should print out the workspace path where all your exercises will be downloaded.

**Comment by Marlene163** [comment]
> @Marlene163 `exercism workspace` command should print out the workspace path where all your exercises will be downloaded.

Yes, that's true, but I'm using the workspace flag which exercism recommended to use after I pasted the command. Anyway the program puts the workspace somewhere I cannot find it.

**Comment by kotp** [comment]
On 22/01/05 02:35AM, Marlene wrote:
> > @Marlene163 `exercism workspace` command should print out the
> > workspace path where all your exercises will be downloaded.
> 
> Yes, that's true, but I'm using the workspace flag which exercism
> recommended to use after I pasted the command. Anyway the program puts
> the workspace somewhere I cannot find it.
> 

Can you show the command as recommended, and the output of the `exercism
workspace` command?

I do not know that I have ever used the workspace flag, and if I have
seen the recommendation I might just be forgetting about it.  So this is
a good way to have a reminder, and show what I might do differently in
this situation, others might have an immediate recognition of the
problem, if shown.
